primitiveMeshGen.py

```diff
@@ -17,6 +17,56 @@ def pol2cart(rad,theta):
   y = rad*np.sin(theta)
   return x,y
 
+#generate shed object
+def shedGen(height, length, theta, width):
+    xinterval = np.cos(theta) * width / 2
+    yinterval = length / 2
+    roof_height = xinterval * np.tan(theta)
+    vertices = []
+    vertices.append([-xinterval, yinterval, roof_height])
```

Comment:
It seems that the assumption here is that the origin is the center of the building roof and the height of the building is extruded downward from this point.  Documentation above would make this more clear.  I think this approach works, but as with the width parameter, I think it makes more sense to define the origin height at the lowest point of the roof plane.  This would be more consistent with other roof types.  For example, this height parameter would be consistent with body_height in the gable roof.